src/features/problems/components/tasks/file-editor.tsx

```diff
@@ -117,7 +120,7 @@ const FileEditor: React.FC<FileEditorProps> = ({
 
   return (
     <div className={cn("flex h-full grow flex-col", className)}>
-      <div className="flex border-b border-[#333] bg-[#1e1e1e]">
+      <div className="flex">
```

Comment:
```suggestion
      <div className="flex bg-background">
```

to fix

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/47d89d45-68dd-46b0-8c47-69efdeb29125" />